server/commands/music.py

```diff
@@ -428,18 +434,18 @@ def ooc_cmd_musiclist_save(client, arg):
             return
 
     if len(args) > 2 and args[2].lower() == "read_only":
```

Comment:
`/save_hub args[0] args[1] args[2]`, args[2] doesn't exist.
this should be `len(args) > 1 and args[1].lower()`, please check all instances of this